pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -142,14 +143,29 @@ func checkLibvirtInstalled() error {
 	return nil
 }
 
-func fixLibvirtInstalled() error {
-	logging.Debug("Trying to install libvirt")
-	stdOut, stdErr, err := crcos.RunWithPrivilege("install virtualization related packages", "yum", "install", "-y", "libvirt", "libvirt-daemon-kvm", "qemu-kvm")
-	if err != nil {
-		return fmt.Errorf("Could not install required packages: %s %v: %s", stdOut, err, stdErr)
+func fixLibvirtInstalled(distro linux.OsType) func() error {
+	return func() error {
+		logging.Debug("Trying to install libvirt")
+		stdOut, stdErr, err := crcos.RunWithPrivilege("install virtualization related packages", "/bin/sh", "-c", installLibvirtCommand(distro))
+		if err != nil {
+			return fmt.Errorf("Could not install required packages: %s %v: %s", stdOut, err, stdErr)
+		}
+		logging.Debug("libvirt was successfully installed")
+		return nil
+	}
+}
+
+func installLibvirtCommand(distro linux.OsType) string {
+	yumCommand := "yum install -y libvirt libvirt-daemon-kvm qemu-kvm"
+	switch distro {
+	case linux.Ubuntu:
+		return "apt-get update && apt-get install -y qemu-kvm libvirt-daemon libvirt-daemon-system dnsmasq"
```

Comment:
On my ubuntu VM, `qemu-kvm` is installed as a dependency of `libvirt-daemon`. Not sure why `dnsmasq` is needed? For virsh, you need to add `libvirt-clients` to that list.